File: PHP/PC.php
<?php
    class PC
    {
        // PC attributes.
        private $processor;
        private $disk;
        private $ram;

        /* Constructor.
           Please take note that after a few try, PHP seems to not accept
           multiple constructor. So I decided to set it as data constructor. */

        // Constructor with PC's data.
        public function __construct($processor, $disk, $ram)
        {
            $this->processor = $processor;
            $this->disk = $disk;
            $this->ram = $ram;
        }

        /* Setter and Getter. */

        // Set processor attribute.
        public function setProcessor($processor)
        {
            $this->processor = $processor;
        }

        // Set disk attribute.
        public function setDisk($disk)
        {
            $this->disk = $disk;
        }

        // Set RAM attribute.
        public function setRAM($ram)
        {
            $this->ram = $ram;
        }

        // Get processor attribute.
        public function getProcessor()
        {
            return $this->processor;
        }

        // Get disk attribute.
        public function getDisk()
        {
            return $this->disk;
        }

        // Get RAM attribute.
        public function getRAM()
        {
            return $this->ram;
        }

        /* Destructor. */

        public function __destruct()
        {
            
        }
    }
?>

File: C++/PC.cpp
#include "header.hh"

/* Constructor. */

// Empty constructor.
PC::PC()
{

}

// Constructor with PC's data.
PC::PC(Processor processor, Disk disk, RAM ram)
{
    this->processor = processor;
    this->disk = disk;
    this->ram = ram;
}

/* Setter and Getter. */

// Set processor attribute.
void PC::setProcessor(Processor processor)
{
    this->processor = processor;
}

// Set disk attribute.
void PC::setDisk(Disk disk)
{
    this->disk = disk;
}

// Set RAM attribute.
void PC::setRAM(RAM ram)
{
    this->ram = ram;
}

// Get processor attribute.
Processor PC::getProcessor()
{
    return processor;
}

// Get disk attribute.
Disk PC::getDisk()
{
    return disk;
}

// Get RAM attribute.
RAM PC::getRAM()
{
    return ram;
}

/* Destructor. */

PC::~PC()
{

}